tests/Ethereum/ContractCallTests.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Ethereum/ContractCall.h"
+#include "HexCoding.h"
+
+#include <fstream>
+#include <gtest/gtest.h>
+
+using namespace TW;
+using namespace TW::Ethereum;
+
+extern std::string TESTS_ROOT;
```

Comment:
Why using extern here?  Where is it set?

Reply:
in gtest main.cpp